README.md

```diff
@@ -105,6 +105,9 @@ Our library supports auto-translation for various user languages. Please read ou
 
 We welcome code changes that improve this library or fix a problem. Please make sure to follow all best practices and add tests, if applicable, before submitting a pull request on GitHub. We are pleased to merge your code into the official repository if it meets a need. Make sure to sign our [Contributor License Agreement (CLA)](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) first. See our license file for more details.
 
+### Using your local `stream-chat-js` for testing
+From the root of this project you can do `yarn add file:./../stream-chat-js` and link your local client js (with changes to test) and this is working also for the examples folders.
+
```

Comment:
Hey @guerinoni, perhaps it would be better if you put a link to our Developer documentation:
- https://github.com/GetStream/stream-chat-react/blob/master/developers/DEVELOPMENT.md